src/js/media-poster-image.js

```diff
@@ -0,0 +1,70 @@
+import { defineCustomElement } from './utils/defineCustomElement.js';
+import {
+  Window as window,
+  Document as document,
+} from './utils/server-safe-globals.js';
+import { MediaUIAttributes } from './constants.js';
+
+const template = document.createElement('template');
+
+template.innerHTML = `
+  <style>
+    :host {
+      height: 100%;
+      width: 100%;
+      position: relative;
```

Comment:
might need `pointer-events: none;` so the gesture layer can still receive mouse events
https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events